lib/screens/home_page.dart

```diff
@@ -199,6 +199,7 @@ class _HomeState extends State<Home> {
                   hintStyle: TextStyle(color: Colors.white)),
             ),
       centerTitle: true,
+      //toolbarHeight: 100,
```

Comment:
How come you dont use `EurekaAppBar()` inside the `Home()`?

Reply:
I assume it's because that `Home()` has unique app bar with search icon and chat button

Reply:
Max is correct there was too many parameters to account for in home. The built in appbar() flutter offers is a better choice, due to the many parameters it can accept. @tonynguyen98